portfolio/src/main/webapp/socials.html

```diff
@@ -12,5 +12,28 @@
       <a class="active" href="socials.html">socials</a>
       <a href="misc.html">misc</a>
     </div>
+    <div>
+      <div id="content">
+        <a class="image-link" href="https://www.linkedin.com/in/aaronlopes/" target="_blank">
+          <img border="0" src="images/linkedin.png">
+        </a>
+        <div class="divider"></div>
+        <a class="image-link" href="https://github.com/AaronLopes/" target="_blank">
+          <img src="images/gh.png">
+        </a>
+        <div class="divider"></div>
+        <a class="image-link" href="https://www.instagram.com/aaron.lopes/" target="_blank">
+          <img src="images/ig.png" alt="instagram">
+        </a>
+        <div class="divider"></div>
+        <a class="image-link" href="https://www.facebook.com/aaron.lopes.378" target="_blank">
+          <img src="images/fb.png" alt="facebook">
+        </a>
+      </div>
+    </div>
+    <div id="content">
```

Comment:
You have multiple HTML elements with id "content", which is technically not allowed - maybe use a class instead?